translations/bnpchs-final-2019-Rbit.html

```diff
@@ -0,0 +1,58 @@
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  In a 1 dimensional coordinate system, Feli has a rabbit named Rbit. Rbit likes to jump a lot. In every jump, Rbit can move to the right or move to the left.
+</p>
+
+<p style="text-align: justify;">
+  Every time Rbit jumps, the jump distance is reduced by 1, if compared to the previous one. If Rbit jumps a 7 long-distance from coordinate 3 to 10, then in the next one it jumps for 6 long-distance ends up in coordinate 4 or 16. As Rbit loves jumping, Rbit will keep jumping until it's jumping distance reach 0. Rbit can also jump into a negative coordinate point.
+</p>
+
+<p style="text-align: justify;">
+  Rbit lives at coordinate point 0 and wants to go outside. The only way out is at the coordinate point $$$N$$$. Help Rbit to get outside by specifying the minimum first jumping distance and the direction of every jump so that Rbit stops right at coordinate $$$N$$$. If Rbit can't go outside output "-1" instead.
```

Comment:
```suggestion
  Rbit lives at coordinate point 0 and wants to go outside. The only way out is at the coordinate point $$$N$$$. Help Rbit to get outside by specifying the minimum first jumping distance and the direction of every jump so that Rbit stops right at coordinate $$$N$$$. If Rbit cannot go outside, output "-1".
```